src/code42cli/cmds/securitydata.py

```diff
@@ -267,9 +262,98 @@ def show(state, search_id):
     echo(pformat(response["searches"]))
 
 
+@security_data.command()
+@file_event_options
+@search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
+@sdk_options()
+@server_options
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display simple properties of the primary level of the nested response.",
+)
+@send_to_options
+def send_to(
+    state,
+    format,
+    hostname,
+    protocol,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    include_all,
+    **kwargs
+):
+    """Send events to the given server address."""
+    output_header = ext.try_get_default_header(
+        include_all, SEARCH_DEFAULT_HEADER, format
+    )
+    format_func = get_file_events_output_format_func(format)
+    send_events_decorator = send_events(
+        format, hostname, protocol, output_header, format_func
+    )
+    handlers = _extract_events(
+        state,
+        begin,
+        end,
+        advanced_query,
+        use_checkpoint,
+        saved_search,
+        or_query,
+        output_function=send_events_decorator,
+        **kwargs,
+    )
+    if not handlers.TOTAL_EVENTS and not errors.ERRORED:
+        echo("No results found.")
+
+
 def _get_file_event_extractor(sdk, handlers):
     return FileEventExtractor(sdk, handlers)
 
 
 def _get_file_event_cursor_store(profile_name):
     return FileEventCursorStore(profile_name)
+
+
+def _extract_events(
+    state,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    output_function,
```

Comment:
See https://github.com/code42/code42cli/pull/137#discussion_r474188218

Instead of taking output_function`, make `send_to()` create different handlers using `extraction.py`. It will be the implementation much more straightforward with less decorators /passing function around.  I will add a comment in `extraction.py` will additional info

Reply:
Done.